app/templates/page-not-found.hbs

```diff
@@ -0,0 +1,5 @@
+{{page-title "Page Not Found"}}
+<div class="page-not-found">
+	<img class="page-not-found__img" src="../assets/images/not-found.png" alt="not-found">
+	<h1 class="page-not-found__text">The page you're looking for cannot be found!</h1>
+</div>
```

Comment:
Can we please fill the page with the remaining space, means on the `scroll: 0` there should be only navbar and 404 content visible, then after scrolling a bit, footer should appear. Refer to the link and screenshot below.

**Link :** https://realdevsquads.vercel.app/join
![Screenshot from 2023-01-30 15-42-44](https://user-images.githubusercontent.com/78433013/215449836-653ad41e-b122-46c3-a5f3-e777746086da.png)
![image](https://user-images.githubusercontent.com/78433013/215449954-22029070-c159-48f5-bb51-f344e3acd654.png)
![image](https://user-images.githubusercontent.com/78433013/215449729-cb823566-f9df-436e-b71f-b1357b36a5d7.png)